members/V000128.yaml

```diff
@@ -3,82 +3,114 @@ contact_form:
   method: post
   action: ""
   steps:
-    - visit: "https://www.vanhollen.senate.gov/content/contact-senator"
-    - find:
-        - selector: form#webform-client-form-78 input#edit-submitted-constituent-information-first-name
+    - visit: "https://www.vanhollen.senate.gov/contact/email"
+    - click_on:
+      - selector: ul#listblocks .opinion a
```

Comment:
this is the same selector as the "Find Assistance, Resources & More" link. if they change the order then that would be the one clicked - could you make the selector unique to the "Share Your Opinion On Legislation" link?

Reply:
I only see one instance of this selector. The "Find Assistance, Resources & More" link is slightly different from the "Share Your Opinion On Legislation" one, in particular in regards to the .opinion selector. See red underline in screenie below 
![image](https://user-images.githubusercontent.com/8680733/38274342-8243a13e-375c-11e8-891e-b6467afca052.png)

Should I still make this selector a little bit more unique just in case?

Reply:
Nope, `.opinion` does it!